File: src/main/java/ordilov/lissn/member/application/port/in/PlayCommand.java
package ordilov.lissn.member.application.port.in;


import ordilov.lissn.member.domain.playing.PlayingCommand;

public interface PlayCommand {
  void changePlaying(PlayingCommand.PlayingPlaylist command);

  void deletePlaying(Long memberId, Long playingId);

  void savePlaying(Long memberId, Long playingId);
}

File: src/main/java/ordilov/lissn/member/application/port/in/PlayCommand.kt
package ordilov.lissn.member.application.port.`in`

import ordilov.lissn.member.domain.playing.PlayingPlaylistDto;

interface PlayCommand {
    fun addPlaying(command: PlayingPlaylistDto)

    fun deletePlaying(memberId: Long, playingId: Long)

    fun playTrack(memberId: Long, playingId: Long)

    fun changePlaying(playingCommand: PlayingPlaylistDto)
}